docs/technical-docs/zotero.md

```diff
@@ -0,0 +1,102 @@
+
+Please note that subcollections/project folders do not actually house the subfolders (Triage, In Process, and Annotated) and are entirely separate folders from the subfolders. Papers and tags found in the Triage folder are not searchable from the main project folder and vice versa. 
+
+### Project folder:
+
+This folder should only contain a Standalone Note (https://www.zotero.org/support/notes) that lists the conceptual tags that are relevant for that project. No papers are saved in this folder. 
+
+### Triage: 
+
+This folder is a staging area for papers. A lab member confident in paper triage will determine if the paper is relevant to the project and worth annotating. If it passes, it gets added to the In Process folder and removed from Triage. If it fails, it gets removed from collection (NOT trashed from library as a whole), and project tag is removed.
+
+### In Process:
+
+This folder is for papers that need to be annotated or are actively in the process of being annotated. Once they have been annotated, they are moved to the Annotated folder.
+
+### Annotated: 
+
+This folder is for papers that have been fully read and annotated for a literature review. Papers in this folder should also have the relevant conceptual tags for the project. 
+
+
+
+## Creating and Navigating Tags
+
+Papers can be assigned tags that sort papers by topic. A project might have tags that refer to theories, concepts, and methods. The tags should be developed by members of the project and listed in a Standalone Note to ensure that all project members are using the same tags. A project's Standalone Note will live in the project's folder.
+
+Certain tags might be utilized for multiple projects. In order to make sure tags stay standardized across the lab, there's a Standalone Note in the main NDC_Lab folder that contains the tags from all of the lab's projects called "NDCLab Conceptual Tags". Refer to this Note before adding new tags to your project.
+
+Papers in a subcollection or in the main NDC_Lab group library can be filtered by clicking tags on and off in the lower left-hand corner of the Zotero app. Clicking multiple tags will narrow your search further. Clicking on an already selected tag will remove it as a search parameter. 
+
+
+
+## Quick Tips
+
+1.	This website has an overview of all of Zotero’s features: https://www.zotero.org/support/quick_start_guide 
+2.	The Chrome extension might not work unless you have the Zotero app open and running on your computer.
+3.	Once you save a pdf to a folder, it should automatically retrieve the metadata from the pdf so it's accessible for citation purposes and so it can be sorted by the author, date published, title, abstract, etc in the Zotero app. If it retrieve do it automatically, right click on a pdf in the app and click Retrieve Metadata. You may have to manually type in the info or correct the info it generates. 
```

Comment:
I think there's something wrong with one of the sentences here?  I don't fully understand.